crates/l2/utils/mod.rs

```diff
@@ -26,14 +25,23 @@ where
     hex.serialize(serializer)
 }
 
-pub fn get_address_from_secret_key(secret_key: &SecretKey) -> Result<Address, TryFromSliceError> {
+pub fn get_address_from_secret_key(secret_key: &SecretKey) -> Result<Address, EthClientError> {
     let public_key = secret_key
         .public_key(secp256k1::SECP256K1)
         .serialize_uncompressed();
     let hash = keccak(&public_key[1..]);
 
     // Get the lat 20 bytes of the hash
```

Comment:
nit: 
```suggestion
    // Get the last 20 bytes of the hash
```

Reply:
fixed in bec14139b55734f5f34037cefff5ff577c151829